core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -238,7 +238,7 @@ <h3 class="e2e-test-joyride-title">Preview</h3>
     text-align: center;
     top: 56px;
     width: 100%;
-    z-index: 990;
+    z-index: 1010;
```

Comment:
Why do we have to change this? Since the toaster message is already above the .navbar-helper.

Reply:
This is done so that the third party component(leaflet) will go behind the .navbar-helper.

Reply:
Thanks @prafulbbandre , Can you please let me know where is z-index set for the leaflet ?

Reply:
It's third party import and i dont find the css file for it but i found from the developer tools that it is set to 1000.
Correct me.